### PR TITLE
Typo

### PR DESCRIPTION
**Pull request by MohamadJaber1**

### CODE REVIEW

**Review by fchollet — COMMENTED** [review]
Thanks for the PR! Please also update the `md` and `ipynb` files with the same fix.

**Review by kerrychu — APPROVED** [review]

File: examples/generative/md/molecule_generation.md [hunk 1]
```diff
@@ -696,8 +696,8 @@ MolsToGridImage(
 ```
 
 
-
-
+    
+
```
Comment (fchollet) [inline examples/generative/md/molecule_generation.md:700]: Please revert whitespace changes

File: examples/generative/md/molecule_generation.md [hunk 2]
```diff
@@ -722,8 +722,8 @@ def plot_latent(vae, data, labels):
 plot_latent(model, [adjacency_tensor[:8000], feature_tensor[:8000]], qed_tensor[:8000])
 ```
 
-
-![png](/img/examples/generative/molecule_generation/molecule_generation_23_0.png)
+    
+![png](/img/examples/generative/molecule_generation/molecule_generation_23_0.png)
```
Comment (fchollet) [inline examples/generative/md/molecule_generation.md:726]: Same here and above

**Review by fchollet — APPROVED** [review]
LGTM

**Pull request merged**